src/components/Header/Header.css

```diff
@@ -1,56 +1,328 @@
-    margin: 10px;
-    box-sizing: content-box;
+  display: grid;
+  grid-template-columns: 3fr;
+  background-color: #242424;
+  border-radius: 14px;
+  position: relative;
+  padding: 140px 100px;
+  margin: 10px;
+  box-sizing: content-box;
 }
 
 .header__main {
-    position: relative;
-    max-width: 330px;
-    margin: 0 auto;
-    display: flex;
-    flex-direction: column;
-    margin-top: 160px;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+  margin-top: 90px;
 }
 
 .header__logo {
-    position: absolute;
-    top: 0;
-    left: 37px;
+  width: 23px;
+  height: 28px;
+  position: absolute;
+  top: 0;
+  left: 37px;
 }
 
 .header__title {
-    color: #fff;
-    font-size: 34px;
-    font-style: normal;
-    font-weight: 700;
-    line-height: 43px;
-    letter-spacing: 0em;
-    text-align: center;
-    margin: 0;
-    padding: 0;
-    padding-top: 28px;
+  color: #fff;
+  font-size: 34px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 43px;
+  letter-spacing: 0em;
+  text-align: center;
+  margin: 0;
+  padding: 0;
+  padding-top: 28px;
 }
 
 .header__subtitle {
-    color: #a0a0a0;
-    font-size: 14px;
-    font-style: normal;
-    font-weight: 400;
-    line-height: 20px;
-    text-align: center;
-    margin-top: 16px;
-    margin-bottom: 90px;
-    padding: 0;
+  max-width: 320px;
+  color: #a0a0a0;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 20px;
+  text-align: center;
+  margin: 16px auto 92px;
+  padding: 0;
 }
 
 .header__buttons {
-    display: flex;
-    gap: 10px;
+  display: flex;
+  gap: 10px;
+}
+
+@keyframes straight-Y {
+  0%,
+  100% {
+    transform: translateY(0);
+  }
+  50% {
+    transform: translateY(-145px);
+  }
+}
+
+@keyframes left-curve-X {
+  0%, 100% {
+    bottom: 30px;
+    left: 156px;
+  }
+  40%, 60% {
+    transform: rotate(0);
+    bottom: 30px;
+    left: 10px;
+  }
+  50% {
+    transform: rotate(-90deg);
+    bottom: -2px;
+    left: -10px;
+  }
+}
+
+@keyframes left-curve-Y {
+  0%, 100% {
+    bottom: 0;
+    left: -14px;
+  }
+  40%, 60% {
+    transform: rotate(0);
+    bottom: 140px;
+    left: -14px;
+  }
+  50% {
+    transform: rotate(90deg);
+    bottom: 166px;
+    left: 14px;
+  }
+}
+
+@keyframes right-curve-X {
+  0%, 100% {
+    bottom: 78px;
+    left: 130px;
+  }
+  20%, 80% {
+    transform: rotate(0);
+    bottom: 78px;
+    left: 15px;
+  }
+  30%, 70% {
+    transform: rotate(-90deg);
+    bottom: 55px;
+    left: -8px;
+  }
+  50% {
+    bottom: 0;
+    left: -8px;
+  }
+}
+
+@keyframes right-curve-Y {
+  0%, 100% {
+    bottom: 0;
+    left: -11px;
+  }
+  20%, 80% {
+    transform: rotate(0);
+    bottom: 140px;
+    left: -11px;
+  }
+  25%, 75% {
+    transform: rotate(90deg);
+    bottom: 166px;
+    left: 14px;
+  }
+  50% {
+    bottom: 166px;
+    left: 125px;
+  }
+}
+
+.header__left-column {
+  position: relative;
+  display: flex;
+}
+
+.header__left-column :first-child {
```

Comment:
should we remove spaces from pseudoclasses? @TanyaKomar

Reply:
I don't know why but it doesn't work without spaces

Reply:
Oh that's interesting. The only usage of a space in css selectors that I'm aware of is when selecting descendants of an element. Maybe you can do something similar with pseudoclasses? 
![image](https://user-images.githubusercontent.com/37159177/112550764-43616800-8d96-11eb-973c-a579a27f6f44.png)